fedimint-api/src/module/mod.rs

```diff
@@ -142,7 +140,7 @@ pub struct ApiEndpoint<M> {
         for<'a> fn(&'a M, serde_json::Value) -> BoxFuture<'a, Result<serde_json::Value, ApiError>>,
 }
 
-#[async_trait(?Send)]
```

Comment:
Do we need `Send`? Seems nice to have but broke WASM somehow? Although I'm wondering what `FederationModule` has to do with WASM :sweat_smile: Maybe we should just not compile the `FederationModule` impls in WASM mode for now. Eventually that will go into the server part of modules, which doesn't need WASM.

Reply:
Bummer. I was hoping to get rid of it, so starting consensus on another thread is easier...

Reply:
I think to make it work you'd need to make a lot of stuff conditionally compiled if the target isn't WASM. Or find a way to make that WASM future `Send`.